beancount_web/util/dateparser.py

```diff
@@ -80,7 +80,7 @@ def parse_date(string):
     where start and end look like one of the above examples
     """
     if string.strip().lower() in ['year to date', 'ytd']:
-        return parse_date(str(datetime.date.today().year) + ' - today')
+        return parse_date(str(datetime.date.today().year) + ' - tomorrow')
```

Comment:
This is the wrong place to "fix" this problem. For our purpose, exclusive dateranges should be returned for all parsed dates, so this needs fixing in the `daterange` function.